pytrip/res/concave_tool.py

```diff
@@ -0,0 +1,306 @@
+import math
+
+# This file contains methods that create from a list of intersections (list of points).

```

Comment:
Convert to a triple-quoted module doctring.
See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

> The docstring for a module should generally list the classes, exceptions and functions (and any other objects) that are exported by the module, with a one-line summary of each. (These summaries generally give less detail than the summary line in the object's docstring.) The docstring for a package (i.e., the docstring of the package's __init__.py module) should also list the modules and subpackages exported by the package.